library/ziti_enroll.c

```diff
@@ -162,8 +158,19 @@ static void well_known_certs_cb(char *base64_encoded_pkcs7, const ziti_error *er
     ZITI_LOG(TRACE, "CA PEM:\n%s", ca);
 
     tls_context *tls = default_tls_context(ca, (strlen(ca) + 1));
+    if (enroll_req->ecfg->private_key != NULL) {
+        ziti_err = ZITI_KEY_LOAD_FAILED;
+        if (load_key_internal(tls, &enroll_req->ecfg->pk, enroll_req->ecfg->private_key) != 0) {
+            ZITI_LOG(WARN, "failed to load private key[%s]", enroll_req->ecfg->private_key);
+            if (enroll_req->ecfg->zej->method == ziti_enrollment_methods.ott &&
+                strncmp(enroll_req->ecfg->private_key, "pkcs11://", strlen("pkcs11://")) == 0) {
+                ZITI_LOG(INFO, "attempting to generate pkcs11 key");
```

Comment:
debug level?

Reply:
I think this deserves INFO -- a new key will be added to user's token